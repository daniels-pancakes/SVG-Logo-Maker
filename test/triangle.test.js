const inquirer = require('inquirer');
const shape = require('./lib/shapes.js');

describe('Shape', () => {
    describe('Triangle', () => {})
        it('should');
});