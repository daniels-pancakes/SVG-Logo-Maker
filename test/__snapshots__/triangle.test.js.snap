// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shape Triangle Render should output code to build an SVG xml file. 1`] = `
"<svg viewBox="0 0 100 100" xmlns="http://www.w3.org/2000/svg">
        <polygon points="50,10 100,90 0,90" fill="shapecolor"/>
        	<text style="font-size:20;" fill="textcolor" text-anchor="middle" dominant-baseline="middle" x="50" y="58" class="text">text</text>
	    <text style="font-size:4; letter-spacing:1.5;" fill="textcolor" text-anchor="middle" dominant-baseline="middle" x="50" y="73" class="text">EST. 2024</text>
        </svg>"
`;
